<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogHistogramme</class>
 <widget class="QDialog" name="DialogHistogramme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Histogramme</string>
  </property>
  <widget class="QGraphicsView" name="histogrammeBaseView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Histogramme de l'image de base :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="actionEgaliser">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Egaliser</string>
   </property>
  </widget>
  <widget class="QPushButton" name="actionFermer">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="histogrammeResultView">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Histogramme de l'image r√©sultat :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
